PeriodogramAnalysis/spectrum_analysis.py

```diff
@@ -0,0 +1,318 @@
+import os
+
+import pathlib
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.integrate import simpson
+from dataclasses import dataclass
+from collections import defaultdict
+from typing import Tuple, Dict, Any, Optional
+
+from miv.core.operator.operator import OperatorMixin
+from miv.core.operator.wrapper import cache_call
+
+
+@dataclass
+class PowerSpectrumAnalysis(OperatorMixin):
+    """
+    A class to perform PSD Analysis using Welch, periodogram and multitaper PSD.
+    The algorithm implementation is inspired from: <https://raphaelvallat.com/bandpower.html>
+
+    Attributes:
+    -----------
+    band_list : list
+        Frequency bands to analyze, default is [[0.5, 4], [4, 8], [8, 12], [12, 30], [30, 100]], i.e. the five common frequency bands in EEG.
```

Comment:
Don't need to mention EEG

```suggestion
        Frequency bands to analyze.
        (default: ((0.5, 4), (4, 8), (8, 12), (12, 30), (30, 100)) )
```

Reply:
Removed